files/etc/profile.d/uroot.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+export PATH="/go/bin:/ubin:$PATH/buildbin:/bbin"
+
+# The IFS lets us force a rehash every time we type a command, so that when we
```

Comment:
we probably need a README in this directory maybe? 
README
The uroot.sh can be included with -files when we integrate a u-root image into an exiting initramfs such as tinycore.